support/Python/requirements.txt

```diff
@@ -17,10 +17,14 @@
 # autocompletion
 click
 h5py >= 3.5.0
+# Humanize: For human-readable output like "2 hours ago"
```

Comment:
I don't find "2 hours ago" very "human-readable". I care about the exact time, not some rounding that somebody else thinks is "good"

Reply:
It looks like this is only used for printing how long ago a job ended/started, not any scientific quantities. I think this is a reasonable use for this package and it looks readable.

Reply:
I found something like "2 hours ago" more useful than showing a timestamp, but I don't feel super strongly about it.

Reply:
My experience with GitHub switching to "X ago" was that it made it really annoying to figure out when anything was done. "2 days ago" leaves a huge window open, while "Apr. 1, 10am" is still easy for me to figure out it was 2 days ago, but I now also now immediately when the event actually occurred. My biggest concern is that with a terminal-based interface I probably (I say probably because I don't know what fancy voodoo magic terminals can do nowadays) can't hover my cursor over the "X ago" to get a detailed number (the fact that I find the hovering annoying is beside the point, I just want an easy way to get the exact number). I guess my take on it is that in my experience doing a bit of estimating to get "X ago" is easier than going the other way around

Reply:
> to make output less clear

It's less precise but arguably more clear, in the sense that I can glance over it quickly and understand how long ago the job finished. That's why you wouldn't print, say, seconds since 1970 because you have no idea what time that is even though it's very precise. For comparison, here's the formatted date vs the "humanized" output:

![Bildschirm­foto 2023-04-03 um 13 33 32](https://user-images.githubusercontent.com/746230/229621559-447ba96d-313f-4f00-8d23-ab1ab32f2d68.png)

![Bildschirm­foto 2023-04-03 um 13 33 51](https://user-images.githubusercontent.com/746230/229621575-97bb63ea-2622-408a-8e0b-a76870710256.png)

Reply:
I think "X hours ago" is okay-ish. But "X days ago" is bad, that's not helpful. "last week" is terrible, borderline useless IMO. I find the date is easier to deal with, on the hours timescale it's not really a big deal. I think "seconds since 1970" is an exaggerated comparison and not really relevant to the discussion (nor is it more precise than what you printed with DATE TIME).

One thing we could do is have a `~/.spectre` file that we read to allow people to customize these sorts of things. (Probably better would be `~/.config/spectre/*`

Reply:
I'll just do timestamps for now. We can figure out nicer formatting later.

Reply:
What I did now is this:

![Bildschirm­foto 2023-04-03 um 13 59 19](https://user-images.githubusercontent.com/746230/229645037-0594f6a9-b86f-4821-8e85-4f036642cff4.png)

For the date part it shows "today", "yesterday", or a date like "Mar 20". If the year is not the current year it will also append the year like "Mar 20 2022". Sound ok?